packages/salesforcedx-vscode-apex/src/oas/generationStrategy/methodByMethodStrategy.ts

```diff
@@ -96,6 +191,42 @@ export class MethodByMethodStrategy extends GenerationStrategy {
   }
 
   generatePromptForMethod(methodName: string): string {
-    return 'to be fine tuned';
+    let input = '';
+    const methodContext = this.methodsContextMap.get(methodName);
+    input += `${prompts.SYSTEM_TAG}\n${prompts['METHOD_BY_METHOD.systemPrompt']}\n${prompts.END_OF_PROMPT_TAG}\n`;
+    input += `${prompts.USER_TAG}\n${prompts.METHOD_BY_METHOD_USER_PROMPT}\n`;
+    input += '\nThis is the Apex method the OpenAPI v3 specification should be generated for:\n```\n';
+    const documentText = fs.readFileSync(new URL(this.metadata.resourceUri.toString()), 'utf8');
```

Comment:
Reading the Apex class for each method is expensive. this should be done in constructor.